src/Documenter.jl

```diff
@@ -395,8 +395,8 @@ function deploydocs(;
 
```

Comment:
Would you be willing, by any chance, to write a comment here with the logic of how we interpret the Travis variables? I think it would make it easier in the future to understand what these checks do exactly, without having to reverse engineer the logic again.